### PR TITLE
Website: More Apply Page Improvements

### PR DESCRIPTION
**Pull request by cchrischen**
### Summary 
Fixes the following features

- [x] Lowered blob z-index to prevent overlapping
- [x] Widened buttons in coffee chat section to full container width
- [x] Updated application events to spring 2025
- [x] Added another bullet point to developer in FAQ
- [x] Maintained scroll position between tabs
- [x] Fixed native and custom accordion arrows in FAQ

### Notion/Figma Link 
[Notion](https://www.notion.so/Idol-Lead-Sync-781de97ad403492b94b420349f829ea5?pvs=4#cd388ffaf03b4c6d9d5ddb5de752155c)

### Test Plan 
Local deployment

### CODE REVIEW

File: new-dti-website/src/app/globals.css [hunk 1]
```diff
@@ -256,6 +256,10 @@ summary {
   padding-right: 1rem;
 }
 
+summary::webkit-details-marker {
```
Comment (JasonMun7) [inline new-dti-website/src/app/globals.css:259]: Nice detail!!

**Review by JasonMun7 — APPROVED** [review]
This looks good to me once the format/Lint checks are all accounted for!

**Review by andrew032011 — APPROVED** [review]

File: new-dti-website/src/app/apply/page.tsx [hunk 2]
```diff
@@ -76,10 +76,13 @@ const ApplyCoffeeChat = () => (
         Feel free to chat with any of us over email, coffee, lunch-we're happy to help!
       </p>
       <div className="flex md:flex-row xs:flex-col gap-3">
-        <a href={config.coffeeChatLink} className="primary-button">
+        <a href={config.coffeeChatLink} className="primary-button xs:w-full md:w-fit">
```
Comment (Bookie0) [inline new-dti-website/src/app/apply/page.tsx:79]:
```suggestion
        <a href={config.coffeeChatLink} className="primary-button xs:w-full md:w-fit text-center">
```

File: new-dti-website/src/app/apply/page.tsx [hunk 3]
```diff
@@ -76,10 +76,13 @@ const ApplyCoffeeChat = () => (
...
           Coffee chat with us
         </a>
-        <a href={config.coffeeChatFormLink} className="secondary-button secondary-button--red">
+        <a
+          href={config.coffeeChatFormLink}
+          className="secondary-button secondary-button--red xs:w-full md:w-fit"
```
Comment (Bookie0) [inline new-dti-website/src/app/apply/page.tsx:84]:
```suggestion
          className="secondary-button secondary-button--red xs:w-full md:w-fit text-center"
```

**Review by Bookie0 — APPROVED** [review]
nice! just a quick little suggestion to center the labels for the coffee chat buttons :)

**Review by andrew032011 — APPROVED** [review]

**Pull request merged**